doc/source/cite.rst

```diff
@@ -14,7 +14,7 @@ or it can be read programatically as follows:
   import euphonic
   from importlib_resources import files
 
-  with open(files(euphonic).joinpath('CITATION.cff')) as fp:
+  with open(files(euphonic)/'CITATION.cff') as fp:
```

Comment:
I would find these more readable with whitespace around them; makes it a bit clearer an operation is going on.